src/QMCHamiltonians/SOECPotential.h

```diff
@@ -89,7 +89,8 @@ class SOECPotential : public OperatorBase
                               const RefVectorWithLeader<TrialWaveFunction>& wf_list,
                               const RefVectorWithLeader<ParticleSet>& p_list,
                               std::optional<ListenerOption<Real>> listeners,
-                              bool keep_grid = false);
+                              bool keep_grid = false, 
+                              bool exact_spin = false);
```

Comment:
This option doesn't exist in the single walker API.
Considering certain array sizes differ when not using exact_spin. I don't feel safe to select using exact_spin or not at the function level. Could you remove this option?
You may access `use_exact_spin_` of the leader instead.

Reply:
should be fixed now